packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -93,7 +94,8 @@ class InkResponse extends StatefulWidget {
     this.borderRadius: BorderRadius.zero,
     this.highlightColor,
     this.splashColor,
-  }) : super(key: key);
+    bool enableFeedback,
+  }) : enableFeedback = enableFeedback ?? true, super(key: key);
```

Comment:
(and then assert that it's not null, and mention in the docs for the constructor that it must be non-null)